mesop/dataclass_utils/dataclass_utils.py

```diff
@@ -125,6 +127,9 @@ def default(self, obj):
     except ImportError:
       pass
 
+    if isinstance(obj, bytes):
+      return {_BYTES_OBJECT_KEY: base64.b64encode(obj).decode('utf-8')} # Encode bytes
```

Comment:
[nit] Same here. We can assume the reader of the code can read what the code is doing. Usually you want to add comments that are not self-explanatory. For example, the reason why something is being done which may not be intuitive to the reader of the code.